clients/admin-ui/src/features/common/TaxonomiesPicker.tsx

```diff
@@ -0,0 +1,103 @@
+import {
```

Comment:
Made a more generic version of the TaxonomyDisplayAndEdit component so I could use it in the dataset page. The idea is that later we can re-use it for D&D or for the TaxonomyDisplayAndEdit component.